builtin/logical/ssh/path_roles.go

```diff
@@ -66,6 +66,7 @@ type sshRole struct {
 	AlgorithmSigner            string            `mapstructure:"algorithm_signer" json:"algorithm_signer"`
 	Version                    int               `mapstructure:"role_version" json:"role_version"`
 	NotBeforeDuration          time.Duration     `mapstructure:"not_before_duration" json:"not_before_duration"`
+	NotBeforeAbsolute          time.Time         `mapstructure:"not_before_absolute" json:"not_before_absolute"`
```

Comment:
So I think `not_before` would be the best field name here:

1. PKI calls the field `not_before_duration`, mirroring what we have here for the other field. 
2. It doesn't have `not_before` at all (and as I mentioned above, should probably be added :D)
3. We have `not_after` and `ttl` in PKI, which to me implies the bare field should be the timestamp variant.

So, let's choose to follow the latter and call this `not_before` in both places.